src/xo-app/vm/tab-stats.js

```diff
@@ -31,6 +31,11 @@ export default injectIntl(
       const { id } = this.props.params
       const { granularity } = this.state
 
+      if (this.props.vm.power_state === 'Halted') {
```

Comment:
Watch for `power_state` change in `componentWillReceiveProps()` (!`Running` → `Running`).